src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -462,7 +465,7 @@ private async Task InitializeWorkersAsync()
                 _startupLogger.LogWarning(e, "Unable to create process registry");
             }
 
-            CreateChannel channelFactory = (languageWorkerConfig, registrations) =>
+            CreateChannel channelFactory = (languageWorkerConfig, registrations, attemptCount) =>
```

Comment:
nit: retryCount?

Reply:
It looks like FunctionLoader and WebJobsScriptHostService use the term "attemptCount" so there's at least some precedence?